src/components/animation/animation.ts

```diff
@@ -73,18 +73,18 @@ export default class SlAnimation extends LitElement {
 
   /** Gets and sets the current animation time. */
   get currentTime(): number {
-    return this.animation?.currentTime || 0;
+    return this.animation?.currentTime ?? 0;
   }
 
   set currentTime(time: number) {
-    if (this.animation) {
+    if (typeof this.animation !== 'undefined') {
```

Comment:
At least to me, I'm not sure what `this.animation` is exactly saying from this line. It could be that `animation` is a boolean toggle, or that it is a string which contains an animation declaration which may be empty, etc. But really it just could be undefined, so I think that making the condition explicit is more clear. This comes from the [@typescript-eslint/strict-boolean-expressions](https://typescript-eslint.io/rules/strict-boolean-expressions/) rule if you'd like to read more.

Reply:
OK, I'll give it a try. If it drives me crazy I'll update it later on.